vncviewer/PlatformPixelBuffer.cxx

```diff
@@ -34,7 +34,7 @@
 #include <rdr/Exception.h>
 
 #include "PlatformPixelBuffer.h"
-#include <malloc.h>
+#include <stdlib.h>
```

Comment:
Please squash this in to the original commit.

Reply:
Will do.